demos/storybook/stories/drawer/within-drawer-layout-rail.stories.ts

```diff
@@ -34,17 +43,16 @@ export const withinDrawerLayoutRail = (): any => ({
                </pxb-drawer-footer>
             </pxb-drawer>
             <div pxb-content>
-                <mat-toolbar [style.backgroundColor]="blue" [style.color]="white" 
-                    style="padding: 0 24px">
+                <mat-toolbar [style.backgroundColor]="blue" [style.color]="white" class="app-bar">
                     <button *ngIf="variant === 'temporary'" mat-icon-button 
                         [style.marginRight.px]="direction() === 'rtl' ? -16 : 16"
                         [style.marginLeft.px]="direction() === 'rtl' ? 16 : -16"
                         (click)="state.open = true">
                         <mat-icon>menu</mat-icon>
                     </button>
-                    <h2>Drawer Layout Demo</h2>
+                    <h2 class="mat-display-1">Drawer Layout Demo</h2>
```

Comment:
I see what's happening. This should not be `mat-display-1`... 

React's H6 should correspond to `.mat-h2` or `.mat-title` in ng Material

Reply:
You got it! It'll be deployed in a minute.